Manage.c

```diff
@@ -812,7 +820,7 @@ int attempt_re_add(int fd, int tfd, struct mddev_dev *dv,
 			}
 			if (bitmap_info->bitmap_offset != 1) {
```

Comment:
I know it's not new code, but... what is the "bitmap_offset", and why is "1" a good value for it?

Reply:
```
sysfs.c:
  228         if (strncmp(buf, "file", 4) == 0)
  229             sra->bitmap_offset = 1;                                                                                                                                               
  230         else if (strncmp(buf, "none", 4) == 0)
  231             sra->bitmap_offset = 0;
  232         else if (buf[0] == '+')
  233             sra->bitmap_offset = strtol(buf+1, NULL, 10);
```

We only support "file" bitmaps with DDF...

Reply:
```
mdadm.h
293:	long			bitmap_offset;	/* 0 == none, 1 == a file */
```

Reply:
Ok, lolgtm.